package.json

```
@@ -39,7 +39,7 @@
     "gulp": "3.9.1",
     "jquery": "3.5.0",
     "less": "2.7.2",
-    "lodash": "4.17.19",
+    "lodash": "^4.17.21",
```

Please use fixed version so we will know where we are at
```suggestion
    "lodash": "4.17.21",
```